plone/app/widgets/dx.py

```diff
@@ -973,24 +962,6 @@ def RelatedItemsFieldWidget(field, request, extra=None):
     return FieldWidget(field, RelatedItemsWidget(request))
 
 
-if HAS_PAC:
-    @adapter(getSpecification(IDXCollection['query']), IFormLayer)
-    @implementer(IFieldWidget)
-    def QueryStringFieldWidget(field, request):
-        return FieldWidget(field, QueryStringWidget(request))
```

Comment:
My concerns about removing plone.app.contenttypes support as mentioned above mainly comes of loosing support for plone.app.contenttypes collections - I think this line - and a IDXCollection import from above - should stay.